Dockerfile

```diff
@@ -16,9 +16,11 @@ ARG RUNC_VERSION=v1.0.0-rc9
 ARG CNI_VERSION=v0.8.3
 ARG CONTAINERD_VERSION=1.3.2
 
-RUN apt update && apt install -y curl
+# make `ctr` target the default concourse namespace
+ENV CONTAINERD_NAMESPACE=concourse
 
 RUN set -x && \
+	apt install -y curl iptables && \
```

Comment:
For posterity: while the `firewall` plugin will use `firewalld` and then only fall back on `iptables` if it's not available, and I guess it's not available because we're running in a container. On a real VM this might use `firewalld`? :thinking: 

I wonder what it uses on K8s?

source: https://github.com/containernetworking/plugins/blob/f5c3d1b1bab6ee72a32702c22ebb0f008580ef2d/plugins/meta/firewall/README.md

Reply:
yeah .. I don't really know :thinking:  I think we can get our answer from https://github.com/containerd/cri - I'll add to the list of next steps :+1:  

my guess is that even within a container, we could get it set up, although that'd require us having that daemon up and running (could be an `ifrit.Runner` taking care of it) - if the perf is really convincing, I can totally see that being a valid approach (as far as I remember the big problem w/ iptables is how it has to reload all of its rules when you load something in, while with firewalld, that's on a "diff by diff" basis)

(btw, just found this: https://firewalld.org/documentation/concepts.html)

Reply:
https://github.com/concourse/concourse/issues/5136